### PR TITLE
Add Histrograms for Energy/Force Uncertainty

### PR DESCRIPTION
**Pull request by PythonFZ**
- [x] allow to set the x / y limits
- [x] tests
- [x] metrics for all histograms?

merge after https://github.com/zincware/IPSuite/pull/160

### CODE REVIEW

**Review by M-R-Schaefer — COMMENTED** [review]
Would a stress uncertainty Histogramm be useful as well?

**Comment by PythonFZ** [comment]
> Would a stress uncertainty Histogramm be useful as well?

Moved to issue.

File: ipsuite/analysis/bin_property.py [hunk 1]
```diff
@@ -39,6 +43,14 @@ def get_labels(self):
     def get_hist(self):
         """Create a pandas dataframe from the given data."""
         labels = self.get_labels()
+
+        self.metrics = {
```
Comment (M-R-Schaefer) [inline ipsuite/analysis/bin_property.py:47]: Great addition!

**Review by M-R-Schaefer — APPROVED** [review]
Looks good!

**Pull request merged**